service/src/test/java/uk/gov/hmcts/reform/fpl/service/config/LookupTestConfig.java

```diff
@@ -2,12 +2,20 @@
 
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
 import uk.gov.hmcts.reform.fpl.config.HmctsCourtLookupConfiguration;
+import uk.gov.hmcts.reform.fpl.config.LocalAuthorityNameLookupConfiguration;
 
 @Configuration
 public class LookupTestConfig {
     @Bean
     public HmctsCourtLookupConfiguration courtLookupConfiguration() {
         return new HmctsCourtLookupConfiguration("example=>Family Court:admin@family-court.com");
     }
+
+    @Primary
```

Comment:
This needs to be changed from `@Primary` to  `@ConditionalOnMissingBean`, overwrites stuff accidentally in the integration tests otherwise